ReactiveXComponent/XComponentApi.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.IO;
+
+namespace ReactiveXComponent
+{
+    public class XComponentApi : IXComponentApi
+    {
+        private bool _disposed;
+        private readonly Stream _file;
+        private readonly IXCSessionFactory _sessionFactory;
+
+        public XComponentApi(Stream file)
```

Comment:
"file" bad naming 
Perhaps xcApiStream is better ?
+ use a private constructor. Add a factory method called create or createFromXcApi